Makefile

```diff
@@ -53,6 +53,10 @@ IMAGE:=$(REGISTRY)/node-problem-detector:$(TAG)
 # support needs libsystemd-dev or libsystemd-journal-dev.
 ENABLE_JOURNALD?=1
 
+ifeq ($(shell uname -s 2>/dev/null), Darwin)
```

Comment:
Can we use `go env` instead and also add windows here?

Reply:
Great idea, I'll update shortly.